Makefile.module

```diff
@@ -11,3 +11,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
We shouldn't need a custom Makefile just for this.

Reply:
Pulled it out